types/alloy/index.d.ts

```diff
@@ -0,0 +1,195 @@
+// Type definitions for npm package Alloy 1.14
+// Project: https://github.com/appcelerator/alloy
+// Definitions by: Axway Appcelerator <https://github.com/appcelerator>
+//                 Mathias Lorenzen <https://github.com/ffMathy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
```

Comment:
Be aware that this means that TS3.6 will be the _min_ required version to use this package

Reply:
What version do you recommend?

Reply:
I settled for `3.0`. I assume that's better?

Reply:
It does mean more people might use it. I can't say how likely that is so I hope I'm not coming off as nit picky.

I think `3.1` had some big breaking changes so that seems like a pretty good general older supported version.